src/dmd/module.h

```diff
@@ -53,6 +53,13 @@ class Package : public ScopeDsymbol
     Module *isPackageMod();
 };
 
+struct QualifiedID
```

Comment:
No Ddoc comment

Reply:
Do you want ddoc comments for the C++ headers?

Reply:
No, sorry, I didn't realize it was in a .h file.